src/Listener/PheanstalkLogListener.php

```diff
@@ -44,35 +42,18 @@ public static function getSubscribedEvents()
         ];
     }
 
-    /**
-     * @param CommandEvent $event
-     * @param string       $eventName
-     */
-    public function onCommand(CommandEvent $event, $eventName)
+    public function onCommand(CommandEvent $event, string $eventName): void
     {
         if (!$this->logger) {
             return;
         }
 
-        $pheanstalk = $event->getPheanstalk();
-        $connection = $pheanstalk->getConnection();
-
-        if (!$connection->isServiceListening()) {
-            $this->logger->warning('Pheanstalk connection isn\'t listening');
-        }
-
-        $pheanstalkName = 'unknown';
-        if ($pheanstalk instanceof PheanstalkProxyInterface) {
-            $pheanstalkName = $pheanstalk->getName();
-        }
-
         $nameExploded = explode('.', $eventName);
 
         $this->logger->info(
             'Pheanstalk command: '.$nameExploded[count($nameExploded) - 1],
             [
                 'payload'    => $event->getPayload(),
-                'pheanstalk' => $pheanstalkName,
```

Comment:
IMHO the pheanstalk name should still be logged. Otherwise it's completely unclear which one is affected. I think there are two possible solutions if you want to avoid the extra getter in `PheanstalkProxyInterface`:
1. The `PheanstalkProxy` still has a name that is given in constructor and passed to the `CommandEvent` which gets a new attribute and a corresponding getter for the name.
2. A method is added to `PheanstalkLocator` to get the name for a given pheanstalk which is then used here.

Reply:
Solution 2. seems to be perfect ;)